src/modules/QtQml/Connections.js

```diff
@@ -0,0 +1,41 @@
+/* Connections */
+registerQmlType({
+  module: 'QtQml',
+  name: 'Connections',
+  versions: /.*/,
+  baseClass: 'QtObject',
+  constructor: function QMLConnections(meta) {
+    callSuper(this, meta);
+
+    createProperty("QtObject", this, "target");
+    createProperty("bool", this, "ignoreUnknownSignals");
+
+    this.target = this.$parent
+
+    var old_target = this.target;
+
+    var connections = {}
```

Comment:
Could you port this to the new module definition syntax? Including properties, etc.

`connections` should probably be `this.$connections` to make setCustomSlot a function in the prototype.